backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -3,28 +3,31 @@ package app.ehrenamtskarte.backend
 import app.ehrenamtskarte.backend.common.database.Database
 import app.ehrenamtskarte.backend.common.webservice.GraphQLHandler
 import app.ehrenamtskarte.backend.common.webservice.WebService
+import app.ehrenamtskarte.backend.config.BackendConfiguration
 import app.ehrenamtskarte.backend.stores.importer.LbeDataImporter
 import com.expediagroup.graphql.extensions.print
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.core.ProgramResult
+import com.github.ajalt.clikt.core.requireObject
 import com.github.ajalt.clikt.core.subcommands
 import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
 import java.io.File
 
 class Entry : CliktCommand() {
-    companion object {
-        fun isProductionEnvironment(): Boolean {
-            return System.getProperty("app.production", "").isNotEmpty()
-        }
-    }
+    private val config by option().file(canBeDir = false, mustBeReadable = true).required()
+    private val projectId by option().required()
```

Comment:
Sadly there is not really a built in functionality of clikt to automatically read properties of data classes, therefore we'd have to manually add options and mapping for all properties to be able to overwrite values of the config. Should I do that? And should I do it for all properties?
I am referring to the following from the issue #531 :
> It might be useful to override config options through the CLI. Maybe our CLI framework offers a feature for this. If not done in this task, please create a new issue.

Reply:
I think it has no high priority. So a ticket with your findings is enough. You can assign low priority.

Reply:
Done in this ticket only for properties that I thought may be configured separately.